include/mra/misc/cache.h

```diff
@@ -0,0 +1,72 @@
+#ifndef MRA_CACHE_H
+#define MRA_CACHE_H
+
+#include "mra/misc/types.h"
+
+namespace mra{
+
+  template<typename valueT, Dimension NDIM>
+  class Cache {
```

Comment:
Why do we have a separate cache if the convolution data is already a cache?

Reply:
Will be used in future. Needed to have overloads where user can either pass an `mra::Key` or level `n` and translation `l`.